code/controllers/subsystem/radioactive_nebula.dm

```diff
@@ -0,0 +1,64 @@
+/// Trait for tracking if something already has the fake irradiation effect, so we don't waste time on effect operations if otherwise unnecessary
+#define TRAIT_RADIOACTIVE_NEBULA_FAKE_IRRADIATED "radioactive_nebula_fake_irradiated"
+
+/// Controls making objects irradiated when Radioactive Nebula is in effect.
+SUBSYSTEM_DEF(radioactive_nebula)
+	name = "Radioactive Nebula"
+	flags = SS_BACKGROUND
+	wait = 30 SECONDS
+
+	VAR_PRIVATE
+		datum/station_trait/nebula/hostile/radiation/radioactive_nebula
+
+/datum/controller/subsystem/radioactive_nebula/Initialize()
+	radioactive_nebula = locate() in SSstation.station_traits
+	if (!radioactive_nebula)
+		can_fire = FALSE
+		return SS_INIT_NO_MESSAGE
+
+	// We don't *really* care that this happens by the time the server is ready to play.
+	ASYNC
+		irradiate_everything()
+
+	// Don't leak that the station trait has been picked
+	return SS_INIT_NO_MESSAGE
```

Comment:
I think you need to make this a different setting value, because  `SS_INIT_NO_NEED` not only silences the `to_chat(world)` message but also the logging for initialization time, which we need to know how expensive a subsystem is at this phase.

Reply:
I'm not sure I get you. The only time logging is the to_chat world stuff (that I can find), which is also the spoiler. You want me to just uhhh, only send it to people with +DEBUG or something?

Reply:
I want this subsystem (and future ones like this) to ONLY suppress the to_chat stuff, NOT the log_world call at line 352. https://github.com/tgstation/tgstation/blob/8bb2626c2be5524f7930f1f7b80c18ec261c4433/code/controllers/master.dm#L340-L352

Reply:
Oh I get ya